sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -132,7 +134,7 @@
      * State checker. If this counter changes during an async call, we have to ignore the result in the callback.
      * Cancelling a database call would be unreliable, and if it's too fast you could still have the callback being called.
      */
-    private int mCurrentState;
```

Comment:
This state check code was created before the introduction of the background looper. How many methods from this class are not called from the background looper?